yarn-project/aztec-nr/aztec/src/note/note_getter.nr

```diff
@@ -117,6 +117,7 @@ unconstrained fn get_note_internal<Note, N>(storage_slot: Field, note_interface:
         [],
         1, // limit
         0, // offset
+        false, // include_nullified - TODO: add a way for get_note to retrieve nullified notes?
```

Comment:
needs an issue?
FWIW - I don't think we need to add here but open to thoughts from others. I don't see a use for it I guess

Reply:
I'll just remove the comment if we're not planning on adding this to `get_note`. I agree that it doesn't feel necessary.